addons/payment_aps/const.py

```diff
@@ -0,0 +1,8 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Mapping of transaction states to APS payment statuses
+# See https://paymentservices-reference.payfort.com/docs/api/build/index.html#statuses
+PAYMENT_STATUS_MAPPING = {
+    'pending': ('19',),
+    'done': ('14',),
+}
```

Comment:
Shouldn't we consider more status code ? :thinking: 
Also, the doc link probably should be this one instead
https://paymentservices-reference.payfort.com/docs/api/build/index.html#transactions-response-codes